impl/sql/sqlhist/sqlhist.go

```diff
@@ -24,11 +24,11 @@ import (
 	"errors"
 	"fmt"
 
+	"golang.org/x/net/context"
+
 	"github.com/google/keytransparency/core/transaction"
 	"github.com/google/keytransparency/core/tree"
 	"github.com/google/keytransparency/core/tree/sparse"
-
-	"golang.org/x/net/context"
```

Comment:
Please keep this import here. The current structure of our imports is:
* Standard packages.
* Key Transparency packages.
* Other packages, i.e., this context.
* Proto packages.